LPC Projects/PWM_Sample/src/PWM_Sample.c

```diff
@@ -11,42 +11,225 @@
 #include "board.h"
 #include <stdio.h>
 
-// TODO: insert other include files here
+/* GPIO pins for encoder interrupt */
+#define PIN_A					2 /* GPIO pin number mapped to PIN_A */
+#define PIN_A_PORT				0 /* GPIO port number mapped to PIN_A */
+#define PIN_B					5 /* GPIO pin number mapped to PIN_B */
+#define PIN_B_PORT				0 /* GPIO port number mapped to PIN_B */
+#define IOCON_PIN_A_ID			IOCON_PIO0_2 /* IOCON pin identifer */
+#define IOCON_PIN_B_ID			IOCON_PIO0_5 /* IOCON pin identifer */
+#define ENCODER_IRQ_HANDLER		PIOINT0_IRQHandler /* GPIO interrupt IRQ function name */
+#define ENCODER_NVIC_NAME		EINT0_IRQn /* GPIO interrupt NVIC interrupt name */
 
-// TODO: insert other definitions and declarations here
-///**
-// * @brief	Handle interrupt from SysTick timer
-// * @return	Nothing
-// */
-void SysTick_Handler(void)
+#include <cr_section_macros.h>
+
+volatile uint8_t last_state;
+volatile long ticks;
+
+volatile int desired_steering_angle = 0;
+//volatile int direction = 1; // +1 forwards, -1 backwards
```

Comment:
What was this for?  If we're not going to use it, please remove it

Reply:
We had a sweep node in here at one point, forgot to take it out